themes.mdx

```diff
@@ -1,31 +1,71 @@
+The themes reports in Monterey AI provide a high-level overview of the most common topics and trends in your feedback data. Each theme is produced by grouping together data with some similar element(s) in the raw text content of the feedback. The report is designed to help you understand the most common themes in your feedback, so you can prioritize your product roadmap and make data-driven decisions.
+
+## Themes Reports
+
+By default, the system will create an ongoing report based on **all** of the feedback data in your workspace. This will be updated daily, if there is enough new volume since the last time it was updated.
+
+You can also create additional, one-time themes reports based on a specific subset of your feedback data. This can be useful if you want to focus on a specific product, feature, or customer segment. To do so, you can navigate to the Snippets list view, apply any filters you'd like, and then click the action menu and then "Create Report". This report will not automatically update, but can be refreshed as needed, if there is sufficient new data matching the reports' filters.
 
-Monterey’s structured report categorizes and groups feedback into similar types and themes. The report is updated automatically with new data weekly.
+<img height="100" src="/images/themes/list-view-create-report.png" />
 
-### Snapshot of Feedback Categories
+# Themes List
 
-At a glance, view the total count of snippets received and delve deeper into the segmented numbers for feature requests, bug reports, questions, and appreciations. This offers a quick understanding of the kind of feedback coming your way.
+The Themes List is the default view when you navigate to "Themes" via the navigation menu. This view shows you a list of themes, each with a brief description of the theme, the volume of feedback that matches the theme, the percentage of total feedback that matches the theme, and the average ARR change associated with the theme.
 
-### Top Themes Bar Chart
+Each theme you see contains feedback with some commonality. For example, if you see a theme called "Users requesting dark mode", it means that the feedback snippets in that theme mention dark mode, or something closely related.
 
-Visualize the pulse of your user feedback with a bar chart highlighting the top 5 prevalent themes. This graphical representation allows you to quickly discern which topics are most resonating with your user base.
+Sometimes themes can catch items that are not directly mentioned in the feedback, but are inferred by the system. For example, if you see a theme called "Users requesting dark mode", it could also contain feedback snippets that mention "night mode" or "light mode".
 
-### Detailed Theme Summaries
+Additionally, these themes by default are at a system-wide standard level of granularity. If you wish to customize this, you can reach out to the Monterey AI team at support@monterey.ai (todo link) and discuss your needs and work with us to tune this to them.
```

Comment:
(todo link)

Reply:
weird, idk why i didn't make that a link while writing... will fix! tyty

Reply:
fix't